### DISCUSSION TITLE
Products/companies that use fp-ts

### DISCUSSION THREAD

**Discussion by ingun37** [gcanti/fp-ts#1592] [Q&A] [answered]
I'm trying to convince my team to use fp-ts. Their concern is that fp-ts is not popular enough to guarantee the safety. I've been using fp-ts for a long time and I had very little issues but that's only anecdotal evidence. I'd appreciate if you guys can let me know some products/companies that uses fp-ts.

**Comment by waynevanson** [comment, marked as answer]
https://github.com/gcanti/fp-ts/issues/1124

This is the thread you're looking for. Grammarly did a talk on it too!
